melodic/distribution.yaml

```diff
@@ -8,6 +8,19 @@ release_platforms:
   ubuntu:
   - bionic
 repositories:
+  ColorTuner:
```

Comment:
Can you please rename this to `jderobot_color_tuner`?  That will make it alphabetize with the rest of the jderobot stuff and be easier to see in the future.